.github/CODEOWNERS

```
@@ -148,7 +148,17 @@
 /helmfile.d/values/networkpolicies/ @elastisys/goto-networking
 /helmfile.d/values/networkpolicy/ @elastisys/goto-networking
 /helmfile.d/values/opensearch/ @elastisys/goto-logging-stack
-/helmfile.d/values/podsecuritypolicies/ @elastisys/goto-access-control
+/helmfile.d/values/podsecuritypolicies/common/falco.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/podsecuritypolicies/common/fluentd.yaml.gotmpl @elastisys/goto-logging-stack
+/helmfile.d/values/podsecuritypolicies/common/ingress-nginx.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/podsecuritypolicies/common/kured.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/podsecuritypolicies/common/monitoring.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/podsecuritypolicies/common/rook-ceph.yaml.gotmpl @elastisys/goto-block-storage
+/helmfile.d/values/podsecuritypolicies/common/velero.yaml.gotmpl @elastisys/goto-backups
+/helmfile.d/values/podsecuritypolicies/service/harbor.yaml.gotmpl @elastisys/goto-container-registry
+/helmfile.d/values/podsecuritypolicies/service/opensearch.yaml.gotmpl @elastisys/goto-logging-stack
+/helmfile.d/values/podsecuritypolicies/service/tekton-pipelines.yaml.gotmpl @elastisys/goto-tekton
```

Do we use this group?
```suggestion
/helmfile.d/values/podsecuritypolicies/service/tekton-pipelines.yaml.gotmpl @elastisys/goto-continuous-delivery 
```

---

Yes, the `goto-tekton` should be the correct handle according to our documentation :)